tests/integration/service_test.go

```diff
@@ -101,4 +102,42 @@ var _ = Describe("odoServiceE2e", func() {
 	// 	})
 	// })
 
+	Context("When working from outside a component dir", func() {
+		JustBeforeEach(func() {
+			project = helper.CreateRandProject()
+			context = helper.CreateNewContext()
+			app = helper.RandString(7)
+			originalDir = helper.Getwd()
+			helper.Chdir(context)
+		})
+		JustAfterEach(func() {
+			helper.DeleteProject(project)
+			helper.DeleteDir(context)
+			helper.Chdir(originalDir)
+		})
+		It("should be able to list services in a given app and project combination", func() {
+			// create a component by copying the example
+			helper.CopyExample(filepath.Join("source", "nodejs"), context)
+			helper.CmdShouldPass("odo", "create", "nodejs", "--app", app, "--project", project)
+
+			// create a service from within a component directory
+			helper.CmdShouldPass("odo", "service", "create", "dh-prometheus-apb", "--plan", "ephemeral",
+				"--app", app, "--project", project,
+			)
+			ocArgs := []string{"get", "serviceinstance", "-o", "name"}
```

Comment:
this should specify namespace `"-n", "project"`

/approve cancel

Reply:
Done.

Reply:
@dharmit You need to add the name space in the same occurrence of step in other scenario of this test file

Reply:
https://github.com/openshift/odo/pull/1920/files#diff-2bf31fdcc572d1ff24f1215e18667e08R58 and